### DISCUSSION TITLE
Add non-consecutive line highlighting when viewing a file

### DISCUSSION THREAD

**Discussion by peter-cardenas-ai** [community/community#39788] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

Reposting from this issue here: https://github.com/isaacs/github/issues/453

When viewing a file from a commit, I want to be able to select multiple ranges of lines instead of only one larger range to highlight only the relevant lines.
For my use case, changing the link to support these multiple ranges, i.e. `https://github.com/<org>/<repo>/blob/<commit>/<file path>#L1-L3,L6` would be sufficient

**Comment by ossanna16** [comment]
Hi there @PeterPCardenas and welcome to our community! Thank you for the feedback!

**Reply by peter-cardenas-ai** [reply]
Thanks for the welcome :) Would really love to see this implemented! I think about this everyday when I share a github link. It also seems there is a lot of interest based on this post: https://github.com/orgs/community/discussions/5022
